src/app/Models/User.php

```diff
@@ -66,4 +56,27 @@ public function unfollow(int $userId)
     {
         return $this->followsAction()->detach($userId);
     }
+
+    /**
+     *  認証ユーザー以外のユーザーを呼び出し
+     *
+     * @param int $authUserId
+     *
+     */
+    public function getAllUsers(int $authUserId)
+    {
+        return $this->where('id', '<>', $authUserId)->paginate(Paginate::NUM_TWEET_PER_PAGE);
+    }
+
+    /**
+     * 認証されていないユーザーの場合400エラーで返す
+     *
+     * @param bool $isAuthUser
+     */
+    public function checkIsAuth(bool $isAuthUser)
```

Comment:
引数の型定義をしているのはGood！
出来れば戻り値の型定義をしてあげた方がいいので、下記みたいな感じで書いてみてー！！
※上でcheckIsAuthはbooleanを返したほうがいいとコメントしているので、戻り値の型もbooleanにしてます！
```suggestion
    public function checkIsAuth(bool $isAuthUser): bool
```

Reply:
そういった戻り値の型定義の仕方があること初めて知りました！設定します！